src/bundle.ts

```diff
@@ -58,9 +59,10 @@ async function bundle(
   assignDefault(options, 'target', 'es2015')
 
   const pkg = await getPackageMeta(cwd)
+  pkg.exports &&= await validateExports(pkg.exports, cwd)
```

Comment:
IMO it shouldn't modify the exports here? pkg.exports should be like the original form pased from ` package.json`, if you need to use the parsed/resolved exports, should create another variable and pass it down

Reply:
Modified to pass down `resolvedWildcardExports` as `ExportCondition | undefined` to `getExportPath`. Thank you!